CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This file is influenced by http://keepachangelog.com/.
 ### Changed
 
 ### Fixed
+ - Node preview layout in editorial
```

Comment:
It might be useful just to include what's happening on this line, just in case there were multiple problems with the layout.

`Node preview layout in editorial no longer wraps metadata header incorrectly` or something